pom.xml

```diff
@@ -71,7 +71,6 @@
                 <!-- Make sure we always turn all warnings on during compilation. -->
                 <plugin>
                     <artifactId>maven-compiler-plugin</artifactId>
-                    <version>2.3.2</version>
```

Comment:
Is this change relevant to the jackson upgrade? Or separate? If they are separate we should move it to separate pull requests.

Reply:
will revert